uiowa/uiowa
# Use filtered_html as default text format (if available for field)

Resolves #2377 

# Description

ckeditor text formats need to be set on a site-wide basis, and exceptions handled through custom code. Previously, minimal was set as the default, and exceptions added to certain fields (uiowa_text_area block field and node body fields). As new content gets added that may benefit from default filtered_html, want to get away from minimal as default, filtered_html as exception approach.

Further, there are not currently instances like the hero block which it's important to default to minimal while allowing filtered_html. This PR reverts to setting the default text format ordering as filtered_html > minimal > full_html > plain based on the field and permissions.

# Testing

- Sync a site.
- Edit content fields and check that the text formats are as expected
  - Add a new basic page and edit--is the body field defaulting to filtered_html?
  - Add a new uiowa_text_area block--is the main text field defaulting to filtered_html?
  - Add a banner (or other which should not allow filtered_html) block--is the excerpt or other text field defaulting to minimal, and not allowing to be switched to filtered_html?
